### PR TITLE
system test for new transaction endpoint

### PR DESCRIPTION
**Pull request by smaulik13**

### CODE REVIEW

File: tests/api_tests/0box_transactions._test.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"time"
+
+	"github.com/0chain/system_test/internal/api/util/test"
+	"github.com/stretchr/testify/require"
+)
+
+func Test0BoxTransactions(testSetup *testing.T) {
+	t := test.NewSystemTest(testSetup)
+
+	t.RunSequentiallyWithTimeout("get paginated transactions list while creating pit id", 5*time.Minute, func(t *test.SystemTest) {
```
Comment (Jayashsatolia403) [inline tests/api_tests/0box_transactions._test.go:14]: Why 5 minutes timeout?

File: tests/api_tests/0box_transactions._test.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		txnData, resp, err := zboxClient.GetTransactionsList(t, "")
+		require.NoError(t, err)
+		require.Equal(t, 200, resp.StatusCode())
+
+		if txnData != nil {
```
Comment (Jayashsatolia403) [inline tests/api_tests/0box_transactions._test.go:19]: We will always have transactions so txnData should never be nil. Return error if it is nil.

File: tests/api_tests/0box_transactions._test.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+			pitId := txnData.PitId
+			_, resp, err := zboxClient.GetTransactionsList(t, pitId)
```
Comment (Jayashsatolia403) [inline tests/api_tests/0box_transactions._test.go:21]: Verify the data returned as well. You can request the same transactions from 0chain by hash and compare the content.

**Review by Jayashsatolia403 — CHANGES_REQUESTED** [review]

File: tests/api_tests/0box_transactions._test.go (hunk 1 above)
Comment (smaulik13) [reply tests/api_tests/0box_transactions._test.go:14]: As once pit id is created it is valid for 5 mins

Comment (Jayashsatolia403) [reply tests/api_tests/0box_transactions._test.go]: It's timeout for the test run. You need to know how much should it take to run.

**Review by Jayashsatolia403 — COMMENTED** [review]
lgtm

**Pull request merged**